modules/client-common/src/constants.ts

```diff
@@ -407,3 +408,13 @@ Object.freeze(Permissions);
 export { Permissions };
 Object.freeze(PermissionIds);
 export { PermissionIds };
+export const IPFS_CID_REGEX =
```

Comment:
The regex only applies to CIDv0. Note, that there is also CIDv1 (https://ipfs-search.readthedocs.io/en/latest/ipfs_datatypes.html#cid)

Reply:
I think is missing some parenthesis, but it should support both v1 and v0

Reply:
Ah, in this case, it would be good to write a test for it.

Reply:
True, I thought i had tests for both cases, my bad